R/broom_wrapper.R

```diff
@@ -148,7 +148,9 @@ add_prediction <- function(df, model_df, conf_int = 0.95, ...){
   # if type.predict argument is not indicated in this function
   # and models have $family$linkinv (basically, glm models have it),
   # both fitted link value column and response value column should appear in the result
-  with_response <- !("type.predict" %in% names(cll)) & !is.null(model_df[["model"]][[1]]$family) & !is.null(model_df[["model"]][[1]]$family$linkinv)
+  with_response <- !("type.predict" %in% names(cll)) &&
+                   any(lapply(model_df$model, function(s) { "family" %in% names(s) })) &&
```

Comment:
Wouldn't it be possible ending up trying to access s$family$linkinv when family is NULL with this logic?